File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] num_list) {
        int size = num_list.length;
        int[] answer = Arrays.copyOf(num_list, size+1);
    
        int last = size-1;
        int a = size-2;
        
        if (num_list[a] < num_list[last]) {
            answer[size] = num_list[last] - num_list[a];
        } else {
            answer[size] = num_list[last] * 2;
        }
                    
        return answer;
    }
}

File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    answer = num_list
    last = num_list[len(num_list) - 1]
    a = num_list[len(num_list) - 2]
    
    if a < last:
        answer.append(last - a)
    else:
        answer.append(last * 2)
    
    return answer
